content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -103,10 +103,11 @@ To prepare to build the engine and projects, choose one of the following build t
 1. Use CMake to build the engine as an SDK, the same as if you installed the engine from an installer tool. The following example shows the `profile` build configuration.
 
     ```shell
-    cmake --build build/linux --target INSTALL --config profile -j <number of parallel build tasks>
```

Comment:
For development profile is the recommended build type, debug is much slower to build and run and release is generally optimized for game releases.    If/when you build with the release config I'm not sure if tools like the editor will get built, what result did you get when you tested this?

Reply:
I didn't run --target INSTALL nor --config release so I don't know. I run with --target editor and --config debug.
I would just like to put a note that other build configurations exist and warning about "-Werror".